README.md

```
@@ -26,7 +27,60 @@ pip install uv
+
+```bash
+python -m app.db.config.create_tables
+```
+
+### 6. Poblar FAQs (opcional)
+
+```bash
+python scripts/populate_faqs.py
+```
+
+### 7. Ejecución
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)